examples/step-50/step-50.cc

```diff
@@ -14,906 +14,1449 @@
+  // both processes assemble the face term between a locally owned and a ghost
+  // cell. This is achieved by setting the
+  // MeshWorker::assemble_ghost_faces_both flag. We need to do this, because
+  // we do not communicate the error estimator contributions here.
+  MeshWorker::mesh_loop(dof_handler.begin_active(),
+                        dof_handler.end(),
+                        cell_worker,
+                        copier,
+                        scratch_data,
+                        copy_data,
+                        MeshWorker::assemble_own_cells |
+                          MeshWorker::assemble_ghost_faces_both |
+                          MeshWorker::assemble_own_interior_faces_once,
+                        nullptr /*boundary_worker*/,
+                        face_worker);
+}
+
+
+// @sect4{<code>LaplaceProblem::refine_grid()</code>}
+
+// We used the cell-wise estimator stored in the vector @p estimate_vector and
+// refine a fixed number of cells (chosen here to roughly double the number of
+// DoFs in each step).
+template <int dim, int degree>
+void LaplaceProblem<dim, degree>::refine_grid()
+{
+  TimerOutput::Scope timing(computing_timer, "Refine grid");
+
+  const double refinement_fraction = 1. / (std::pow(2.0, dim) - 1.);
+  parallel::distributed::GridRefinement::refine_and_coarsen_fixed_number(
+    triangulation, estimate_vector, refinement_fraction, 0.0);
+
+  triangulation.execute_coarsening_and_refinement();
+}
+
+
+// @sect4{<code>LaplaceProblem::output_results()</code>}
+
+// The output_results() function is similar to the ones found in many of the
+// tutorials (see step-40 for example).
+template <int dim, int degree>
+void LaplaceProblem<dim, degree>::output_results(const unsigned int cycle)
+{
+  TimerOutput::Scope timing(computing_timer, "Output results");
+
+  VectorType temp_solution;
+  temp_solution.reinit(locally_owned_set,
+                       locally_relevant_set,
+                       mpi_communicator);
+  temp_solution = solution;
+
+  DataOut<dim> data_out;
+  data_out.attach_dof_handler(dof_handler);
+  data_out.add_data_vector(temp_solution, "solution");
+  Vector<float> subdomain(triangulation.n_active_cells());
+  for (unsigned int i = 0; i < subdomain.size(); ++i)
+    subdomain(i) = triangulation.locally_owned_subdomain();
+  data_out.add_data_vector(subdomain, "subdomain");
+
+  Vector<float> level(triangulation.n_active_cells());
+  for (const auto &cell : triangulation.active_cell_iterators())
+    level(cell->active_cell_index()) = cell->level();
+  data_out.add_data_vector(level, "level");
+
+  if (estimate_vector.size() > 0)
+    data_out.add_data_vector(estimate_vector, "estimator");
+
+  data_out.build_patches(0);
+
+  const std::string master = data_out.write_vtu_with_pvtu_record(
+    "", "solution", cycle, mpi_communicator, 2 /*n_digits*/, 1 /*n_groups*/);
+
+  pcout << "   Wrote " << master << std::endl;
+}
+
+
+// @sect4{<code>LaplaceProblem::run()</code>}
```

Comment:
For some reason, your locally compiled version tries to link this to `WorkStream::run()`. I think you want to escape `run`:
```suggestion
// @sect4{<code>LaplaceProblem::%run()</code>}
```